test/views/declaration/destinationCountries/ChangeRoutingCountryViewSpec.scala

```diff
@@ -56,12 +56,12 @@ class ChangeRoutingCountryViewSpec extends UnitViewSpec with Stubs with ExportsT
 
       s"have page question during changing first routing country for ${request.declarationType}" in {
 
-        firstRoutingView(request).getElementsByClass("govuk-fieldset__legend").text() mustBe messages("declaration.firstRoutingCountry.question")
+        firstRoutingView(request).getElementsByTag("h1") must containMessageForElements("declaration.firstRoutingCountry.question")
```

Comment:
No longer has fieldset (not needed for single question/field) - question is in the page heading.